provisioner/powershell/provisioner_acc_test.go

```diff
@@ -44,11 +44,14 @@ func (s *PowershellProvisionerAccTest) GetConfig() (string, error) {
 	filePath := filepath.Join("./test-fixtures", s.ConfigName)
 	config, err := os.Open(filePath)
 	if err != nil {
-		return "", fmt.Errorf("Expected to find %s", filePath)
+		return "", fmt.Errorf("os.Open:%v", err)
 	}
 	defer config.Close()
 
 	file, err := ioutil.ReadAll(config)
+	if err != nil {
```

Comment:
hmm... this should've been caught by the ci-lint check when the file was created. I need to look into why it was not caught. Thanks for make the fix.